ui/src/treeDataProvider/GenericTreeItem.ts

```diff
@@ -17,5 +17,7 @@ export class GenericTreeItem<T = ISubscriptionRoot> extends AzureTreeItem<T> imp
         this.contextValue = treeItem.contextValue;
```

Comment:
I don't understand - what's the treeItem being passed in?

Reply:
Renamed to 'options'